TACC/Core-CMS
# Support custom apps through config.

## Overview
Add support for custom apps through `custom_app_settings.py` and `urls_custom.py`

## Related

- [TUP-250](https://jira.tacc.utexas.edu/browse/TUP-250)

## Changes

- Add configurable settings to insert new apps, middleware and static file paths into the core CMS.

## Testing

1. Copy `custom_app_settings.example.py` into `custom_app_settings.py` adn `urls_custom.example.py` into `urls_custom.py`
2. Start the CMS according to the README instructions and navigate to http://localhost:8000/test/. You should see the `custom_example` app being rendered.

## Screenshots

<img width="1679" alt="image" src="https://user-images.githubusercontent.com/12601812/180084497-336abf56-54f0-4f79-a5f3-9e8526398488.png">
